ModelCatalogueCorePlugin/grails-app/assets/javascripts/modelcatalogue/core/orderedMapEnhancer.coffee

```diff
@@ -15,8 +15,7 @@ angular.module('mc.core.orderedMapEnhancer', ['mc.util.enhance']).config ['enhan
             if value.key == key
               value.value = newValue
               return newValue
-          # only push if the value is defined
-          self.values.push(key: key, value: newValue) if newValue or not key
+          self.values.push(key: key, value: newValue)
```

Comment:
Aha, now I see when the test is failing. Same directive is used for metadata and if I delete this if then it adds meta data like min max, etc...Am going to rollback this.